### ISSUE TITLE
Add an Item - Tags - Adding a new tag lacks status message

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1126]
"Description: On the 'Add an Item' page is a 'Tags' tab. On this page users are able to add tags to the item by entering text into the 'Add Tags' edit field and then selecting the 'Add Tags' button. These tags appear on the screen after they've been added below an 'All Tags' heading. However, there is currently no status message announced to the users when the new tag appears on the page in a new section of content. 

Expected Results: When new information appears on-screen without receiving focus, there is a status message to alert the user of that information. In this case, something like 'Tag Added' or similar would let users know about the successful confirmation.

Actual Results: When users add a new tag from the 'Add Tag' field, there is no status message to alert users of the successful addition."

4.1.3 Status Messages; Level AA; WCAG 2.1

**Comment by aviannamiller** [comment]
Will be fixed in overarching issue (https://github.com/omeka/Omeka/issues/1089)
